solr/CHANGES.txt

```diff
@@ -183,6 +183,10 @@ Upgrade Notes
 
 * SOLR-14065: VelocityResponseWriter has been deprecated and may be removed in a future version.
 
+* SOLR-14072: The "Blob Store" API and "runtimeLib" plugin mechanism that uses it is now considered deprecated.  The
+  replacement to it is the "Package Manager" system, which includes a "File Store".  These are experimental currently
```

Comment:
Lets call it "package store".

Reply:
Should we call it Package Management system? Also, I think there are two spaces before "These".